tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -65,6 +65,7 @@ Define {
 	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer Port0'
 	SMART_AMP_UUID			"1e:96:7a:16:e4:8a:ea:11:89:f1:00:0c:29:ce:16:35"
+	SSP1_ENABLED			"true"
```

Comment:
that's a nice change.

I would flip the logic and make all SSPs disabled by default, so that we can select which ones we want for each platform. This would be very handy when we want to test e.g. on Up Extreme where this only one SSP available on the connector.